Scripts/RemediationScripts/Remediate-ConfigureWAFOnAzureFrontDoorCDN.ps1

```diff
@@ -0,0 +1,795 @@
+﻿<###
+# Overview:
+    This script is used to configure WAF Policy on all endpoints of Front Door CDN(s) in a Subscription.
+
+# Control ID:
+    Azure_FrontDoor_CDNProfile_NetSec_Enable_WAF_Configuration
+
+# Display Name:
+   Front Door CDN should have Web Application Firewall configured
+
+# Prerequisites:
+    1. Contributor or higher privileges on the Front Door CDNs in a Subscription.
+    2. Must be connected to Azure with an authenticated account.
```

Comment:
Can we say Must be connected to Azure with same authenticated account with which have required access over Front door CDNs.